src/sentry_value.c

```diff
@@ -431,6 +431,34 @@ sentry__value_new_object_with_size(size_t size)
     }
 }
 
+sentry_value_t
+sentry_value_new_user(const int32_t *id, const char *username,
```

Comment:
Please also provide a `new_user_n` implementation and implement non-`_n` in terms of `_n'.